Index: routes/web.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- routes/web.php	(date 1524107807000)
+++ routes/web.php	(date 1524107807000)
@@ -18,3 +18,26 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+Route::get('/user/{user_id}/profile', 'ProfileController@create')->name('profile.create');
+Route::get('/user/{user_id}/profile/{profile_id}', 'ProfileController@show')->name('profile.show');
+Route::get('/user/{user_id}/profile/{profile_id}/edit', 'ProfileController@edit')->name('profile.edit');
+Route::post('/user/{user_id}/profile/', 'ProfileController@store')->name('profile.store');
+Route::patch('/user/{user_id}/profile/{profile_id}', 'ProfileController@update')->name('profile.update');
+Route::delete('/user/{user_id}/profile/{profile_id}', 'ProfileController@destroy')->name('profile.destroy');
+
+Route::get('/questions/{question_id}/answers/create', 'AnswerController@create')->name('answers.create');
+
+Route::get('/questions/{question_id}/answers/{answer_id}', 'AnswerController@show')->name('answers.show');
+Route::get('/questions/{question_id}/answers/{answer_id}/edit', 'AnswerController@edit')->name('answers.edit');
+Route::post('/questions/{question_id}/answers/', 'AnswerController@store')->name('answers.store');
+Route::patch('/questions/{question_id}/answer/{answer_id}', 'AnswerController@update')->name('answers.update');
+Route::delete('/questions/{question_id}/answer/{answer_id}', 'AnswerController@destroy')->name('answers.destroy');
+
+
+Route::resources([
+    'questions' => 'QuestionController',
+]);
+
+
+
